docs/current/README.md

```diff
@@ -16,21 +17,49 @@
             </style>
 </head><article class="markdown-body">
 
-# Vaquero
 
-[Home](https://ciscocloud.github.io/vaquero-docs/) | [Docs Repo](https://github.com/CiscoCloud/vaquero-docs/tree/master) | [Project Requirements](requirements.html)
+<div align="center">
+<img src="cow.png" alt="Drawing" style="width: 200px;"/>
+  <p style="font-size:60px">vaquero</p>
+[Home](https://ciscocloud.github.io/vaquero-docs/) | [Dev Repo](https://github.com/CiscoCloud/vaquero) | [Docs Repo](https://github.com/CiscoCloud/vaquero-docs/tree/master) | [Project Requirements](requirements.html) | [Issue Tracking](https://waffle.io/CiscoCloud/vaquero)
+</div>
 
-[![Build Status](https://drone.projectshipped.io/api/badges/CiscoCloud/vaquero/status.svg)](https://drone.projectshipped.io/CiscoCloud/vaquero)
-
-- [Private Dev Repo](https://github.com/CiscoCloud/vaquero) : Vaquero's development home
-- [Waffle.io Issue Tracking](https://waffle.io/CiscoCloud/vaquero): Progress tracking tool
+<h1></h1>
 
 A bare metal configuration utility that network boots machines based on user defined templates. We leverage iPXE and support cloud-config, ignition, kickstart, and untyped unattend boot scripts.
 
 The only thing you need pre-installed to run Vaquero is [Docker](https://www.docker.com/).
 
 See the [Getting Started](getting-started.html) page for details on deploying Vaquero in virtualbox.
 
+## Features
+
+(last update: 11/16/16)
+
+### Completed
+1. A **powerful, customizable** configuration file for your bare metal deployment, with built-in validation tools and examples
```

Comment:
maybe something about single container for simplicity that runs in multiple modes